src/e2e-test/java/io/cdap/plugin/bigqueryexecute/runner/TestRunner.java

```diff
@@ -25,7 +25,8 @@
 @RunWith(Cucumber.class)
 @CucumberOptions(
   features = {"src/e2e-test/features"},
-  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign"},
+  glue = {"io.cdap.plugin.bigqueryexecute.stepsdesign", "io.cdap.plugin.common.stepsdesign", "stepsdesign",
+    "io.cdap.plugin.bigquery.stepsdesign"},
```

Comment:
Why is the big query related step design added? I don't think this is required.

Reply:
This is required because we are doing validation step for the above scenario, validation for the content of records transferred